crates/stress-test/src/main.rs

```diff
@@ -222,11 +241,65 @@ async fn run_stress_test(cli: Cli, c8d: impl Containerd) -> Result<()> {
 
     let elapsed = start.get().unwrap().elapsed();
     let throuput = count as f64 / elapsed.as_secs_f64();
-    let elapsed = format_duration(elapsed);
+    let duration = format_duration(elapsed);
 
     println!("\x1b[32m{success} tasks succeeded\x1b[0m");
-    println!("\x1b[32m  elapsed time: {elapsed}\x1b[0m");
+    println!("\x1b[32m  elapsed time: {duration}\x1b[0m");
     println!("\x1b[32m  throuput: {throuput} tasks/s\x1b[0m");
 
+    let re = Regex::new(r"containerd-shim-(\w+)").unwrap();
+    let shim_path_str = shim_path.to_string_lossy();
+    let shim = re
+        .captures(&shim_path_str)
+        .and_then(|c| c.get(1))
+        .map(|m| m.as_str())
+        .unwrap_or("wasmtime");
+
+    let containerd_shim = if containerd { "containerd" } else { "mock" };
+
+    if let Some(json_output) = json_output {
+        let results = vec![
+            BenchmarkResult {
+                name: format!(
+                    "Stress Test Tasks Throughput with {} service - {}",
+                    containerd_shim, shim
+                ),
+                unit: "tasks/s".to_string(),
+                value: throuput,
+                extra: Some(format!(
+                    "Image: {}\nTasks: {}\nParallel: {}\nDuration: {}",
+                    image, count, parallel, duration
+                )),
+            },
+            BenchmarkResult {
+                name: format!(
+                    "Stress Test Tasks Elapsed Time with {} service - {}",
+                    containerd_shim, shim
+                ),
+                unit: "s".to_string(),
+                value: elapsed.as_secs_f64(),
```

Comment:
This is passed to the benchmark tool in CI as `customBiggerIsBetter`, but this is not the case. 
Personally I wouldn't bother capturing this value as it's implicitly captured in the throughput.